WalletWasabi/Legal/Assets/LegalDocumentsGingerWallet.txt

```diff
@@ -0,0 +1,484 @@
+
+
+2 THE COINJOIN COORDINATOR SERVICES
+
+===================================
+
+
+2.1 DESCRIPTION OF SERVICES
+
+    This Software consists of the Client Application and the CoinJoin Coordinator (both as defined below) and function as a free, open-source, non-custodial desktop wallet. The Software does not constitute an account where We or other third parties serve as financial intermediaries or custodians of Your Bitcoins.
+
+The Software and Our Services are therefore not a bank, a custodian, an exchange, a financial intermediary, or a regulated financial institution.
+
+
+    The client application (“Client Application”) is software, with the sole purpose of allowing You access to the Bitcoin network and our Services (as defined below), without the need or requirement to create or maintain a user account. The application itself is an ordinary Bitcoin desktop wallet with ordinary features.
+
+
+Ginger Wallet does not store, send, or receive Bitcoins. Bitcoins exist only by the ownership record maintained in the Bitcoin network. Any transfer of title in Bitcoins occurs within the decentralized Bitcoin network and not in the Ginger Wallet.
+
+
+
+  The CoinJoin Coordinator Service is an online service that implements trustless CoinJoin to prevent third parties from spying on the Blockchain. Throughout the process, the Service does not initiate or process any standalone transactions towards third parties (i.e. non-users of the Service) and, therefore, does not store or transmit value belonging to others. For further information on how it works please visit Our GitHub page: https://github.com/GingerPrivacy. You acknowledge that the CoinJoin Service runs in the background by default while using the Client Application. To avoid fees, You may turn automatic CoinJoin Services off in the settings of the Client Application.
+
+    The UTXO is an unspent transaction output, the technical term for the amount of digital currency that remains after a cryptocurrency transaction.
+
+    The One Hop transaction is when there is one non-CoinJoin transaction spending only CoinJoin outputs.
+
+2.2 FEES
+
+    Subject to the other provisions of these Terms, including, but not limited to Section 3 regarding Prohibited Activities, You may freely download and use the Client Application without any charge or fee imposed directly on You by the Service Provider.
+
+
+    The Service Provider does not charge You transaction fees for normal Bitcoin transactions, however, You are still subject to transaction fees, charged by the Bitcoin network.
+
+
+    The Service Provider and the Bitcoin network charge You transaction fees for CoinJoin Bitcoin transactions.
+
+
+    The CoinJoin fee, charged by the Service Provider is 0.3% per new UTXO, except if:
+
+     -The UTXO comes directly from a Wasabi Wallet 2.x CoinJoin transaction used zkSNACKs’ coordinator till Jun 1, 2024.
+
+     -The UTXO comes from a One Hop transaction.
```

Comment:
+ The UTXO comes directly from a Ginger Wallet CoinJoin transaction used InvisibleBit LLC. coordinator.

Reply:
The original text was to accept coinjoin from Wasabi 1.0.
I thought that the line you mention comes from the One Hop line.
Should I add your line as well?

Reply:
yes, remixes are free, is that declared elsewhere?

Reply:
> yes, remixes are free, is that declared elsewhere?

I don't see that.

Also shouldn't we mention that it is free under 0.01 BTC?